### DISCUSSION TITLE
Can we get a working pointer within CEMU?

### DISCUSSION THREAD

**Discussion by Dadoum** [v1993/linuxmotehook2#24] [Q&A] [answered]
And if so, how? (I have a working sensor bar in Dolphin x64 Linux, and I am running it from a Fedora 40 machine CEMU Linux native builds)

**Comment by v1993** [comment, marked as answer]
Unfortunately, not possible with this project (or, more generally speaking, with DSU protocol in general), unless Cemu can emulate pointer from gyro. I think they support wiimotes on Linux natively these days, given you configure HIDAPI access? I haven't used it in forever.
